taskcluster/fenix_taskgraph/worker_types.py

```diff
@@ -157,3 +157,56 @@ def build_github_release_payload(config, task, task_def):
         "{}:github:project:{}".format(scope_prefix, worker["github-project"]),
         "{}:github:action:{}".format(scope_prefix, worker["action"]),
     ])
+
+
+@payload_builder(
+    "scriptworker-tree",
+    schema={
+        # FIXME: do we need upstream artifacts?
+        Required("upstream-artifacts"): [
+            {
+                Required("taskId"): taskref_or_string,
+                Required("taskType"): text_type,
+                Required("paths"): [text_type],
+            }
+        ],
+        Optional("dontbuild"): bool,
+        Required("tags"): [Any("buildN", "release", None)],
+        Required("bump"): bool,
+        Optional("bump-files"): [text_type],
+        Optional("push"): bool,
+        # TODO: configure with proper payload expected input
+    },
+)
+def build_version_bump_payload(config, task, task_def):
+    worker = task["worker"]
+    task_def["tags"]["worker-implementation"] = "scriptworker"
+
+    task_def['payload'] = {'actions': []}
+    actions = task_def['payload']['actions']
+    if worker['tags']:
+        # TODO: add tags logic here
```

Comment:
I don't think we need to support tags, at the moment. Githubscript implicitly creates one whenever a github release is made.